### ISSUE TITLE
U+11CA9 MARCHEN SUBJOINED LETTER YA blocks mark positioning

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1725]
### Font
NotoSansMarchen-Regular.otf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/7f12f3e70af9d2507206093d903eb9945d7e4622/phaseIII_only/unhinted/otf/NotoSansMarchen/NotoSansMarchen-Regular.otf
Date: 2020-04-28

### Font version
Version 3.000

### Issue
U+11CA9 MARCHEN SUBJOINED LETTER YA blocks the positioning of subsequent marks.

### Character data
𑱺𑲩𑲶
U+11C7A MARCHEN LETTER TA
U+11CA9 MARCHEN SUBJOINED LETTER YA
U+11CB6 MARCHEN SIGN CANDRABINDU

### Screenshot
![𑱺𑲩𑲶](https://user-images.githubusercontent.com/1124347/80493195-16f95d80-8933-11ea-85bd-9a44b4ab3e38.png)

**Comment by nizarsq** [comment]
@marekjez86 A side note, According to http://www.unicode.org/L2/L2013/13197-marchen.pdf (page 13) U+11CA9 YA is a right-side subjoined letter, is U+11CB6  can come after U+11CA9?   
<img width="1261" alt="Screen Shot 2020-09-12 at 6 44 32 PM" src="https://user-images.githubusercontent.com/22158882/93008302-01728200-f528-11ea-9ea8-359d414201d4.png">

**Comment by punchcutter** [comment]
Yes, U+11CB6 can come after U+11CA9 and in this font that usually happens with ligatures, but the cause of this problem is that U+11CA9 didn't get classified as a mark in the GDEF. I thought I had already submitted a PR for that fix months ago, but I guess not. I just submitted one now. 

<img width="128" alt="marchen1725" src="https://user-images.githubusercontent.com/11257204/93009071-13a4ee00-f531-11ea-9ee1-25ccd0c0df07.png">
